pymon/main.py

```diff
@@ -46,7 +47,7 @@ def handle_event(event):
     observer.schedule(event_handler, os.getcwd(), recursive=True)
 
     observer.start()
-
+    
```

Comment:
More unnecessary spaces

Reply:
Removed extra spaces.
Request and review more changes if needed